frontend/src/components/pages/CampsList/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import CampsNavigationHeading from "./CampsNavigationHeading";
+
+const CampsListPage = (): React.ReactElement => {
+  const [year, setYear] = useState(2022);
```

Comment:
nit: let's use the year from the current date
```suggestion
  const [year, setYear] = useState(new Date().getFullYear());
```